tests-ng/test_packages.py

```diff
@@ -0,0 +1,22 @@
+import pytest
+from plugins.dpkg import Dpkg
+from plugins.shell import ShellRunner
+
+denylist = [
+    "rlogin",
+    "rsh",
+    "rcp",
+    "telnet",
+    "libdb5.3:amd64",
+    "libdb5.3:arm64",
```

Comment:
We should consider if we can improve this, rather then having a hardcoded list, just do the cross product of the package names and all architects returned by `dpkg --print-architecture` and `dpkg --print-foreign-architectures`